src/sidebar/service-url.js

```diff
@@ -3,44 +3,91 @@
 var urlUtil = require('./util/url-util');
 
 /**
- * A map of all route names to relative URLs on the Hypothesis service that
- * the client links to.
+ * An AngularJS service that returns an absolute URL given a link name and
```

Comment:
Hmm. It is using AngularJS dependency injection, although I realise the caller can just pass in `annotationUI` and `store` directly. It's also a little weird that this docstring doesn't actually document the function that it's on, which is a function that returns a function, rather it documents the function that it returns. This feels like an Angular-specific pattern to me (although again, yes, it is just a plain function that returns a function that...). Is this a common pattern, called a "service", that exists beyond just AngularJS?

My other concern is overloading of the word "service" in AngularJS service and the web service and serviceUrl, that's why I said "AngularJS" service to be unambiguous.